### DISCUSSION TITLE
FDroid build for 2024.01.09-5-android still hasn't been released after nearly a month

### DISCUSSION THREAD

**Discussion by secretServant** [organicmaps/organicmaps#7289] [Q&A] [answered]
Strange, the only reason I'm tracking this at all is because 2023.12.20-4-android introduced #6921.

**Comment by Jean-BaptisteC** [comment, marked as answer]
More information here #7218

**Reply by secretServant** [reply]
Jeez, might need to fork the project just for an F-droid release
